setup_go.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex
+
+# Install gimme to get go1.15.11
+curl -sL -o /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
```

Comment:
Is there any way to use a stable version with a hash check of this file?

Reply:
I can add a check for this (I copypasted what was in the `Dockerfile` of the builder before).